embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/ApiService.kt

```diff
@@ -15,5 +15,5 @@ internal interface ApiService {
     fun sendEventAndWait(eventMessage: EventMessage)
     fun sendCrash(crash: EventMessage)
     fun sendAEIBlob(blobMessage: BlobMessage)
-    fun sendSession(sessionPayload: ByteArray, onFinish: (() -> Unit)?): Future<*>
+    fun sendSession(sessionPayload: ByteArray, onFinish: (() -> Unit)?): Future<*>?
```

Comment:
If the request can't be sent return a null Future and have the caller deal with it